packages/starlight/schemas/expressiveCode.ts

```diff
@@ -0,0 +1,11 @@
+import { z } from 'astro/zod';
+import type { StarlightExpressiveCodeOptions } from '../integrations/expressive-code';
+
+export const ExpressiveCodeSchema = () =>
+	z
+		.union([
+			z.custom<StarlightExpressiveCodeOptions>((value) => typeof value === 'object' && value),
```

Comment:
Is the configuration passed down to `astro-expressive-code` validated using Zod? 

If yes, could the schema be exported from it so we can build upon it here instead of using `z.custom` and then infer `StarlightExpressiveCodeOptions` from it in order to provide a consistent experience regarding configuration validation?

Reply:
No, I couldn't justify the (unfortunately) massive footprint of Zod, so I'm not using it in Expressive Code although I would have loved to.

Reply:
My problem here is that Expressive Code is not just for Astro, all the layers below `astro-expressive-code` (where the majority of the config actually comes from) have no Astro dependency at all.